fdbclient/Schemas.cpp

```diff
@@ -1035,3 +1035,16 @@ const KeyRef JSONSchemas::managementApiErrorSchema = LiteralStringRef(R"""(
    "message": "The reason of the error"
 }
 )""");
+
+const KeyRef JSONSchemas::clientLibMetadataSchema = LiteralStringRef(R"""(
+{
+    "platform": "x86_64-linux",
+    "version": "7.1.0",
+    "githash": "e28fef6264d05ab0c9488238022d1ee885a30bea",
+    "type": "debug",
+    "checksum": "fcef53fb4ae86d2c4fff4dc17c7e5d08",
+    "filename": "libfdb_c.7.1.0.so",
+    "size" : 19467552,
+    "chunkcnt" : 2377,
```

Comment:
nit: Given this is the only abbreviation and it's two characters, I would probably use the full word. I don't feel super strongly about this though.

```suggestion
    "chunkcount" : 2377,
```